src/polymer-ide.ts

```diff
@@ -61,21 +68,142 @@ class PolymerIde {
       this.linter.editorService = this.editorService;
       this.autocompleter.editorService = this.editorService;
     }
-  }
+  };
 
   serialize(): ViewState {
     return {};
   };
 
   provideLinter(): lint.Provider|lint.Provider[] {
     return this.linter;
-  }
+  };
 
   provideAutocompleter(): autocomplete.Provider|autocomplete.Provider[] {
     return this.autocompleter;
-  }
+  };
 };
 
+class TooltipManager {
```

Comment:
We should split this off into its own file (likewise for linter and autocompleter, but that's outside the scope of this PR)